bgpd/bgp_packet.c

```diff
@@ -650,7 +651,7 @@ struct stream *bgp_open_make(struct peer *peer, uint16_t send_holdtime, as_t loc
 	stream_putw(s, (local_as <= BGP_AS_MAX) ? (uint16_t)local_as
 						: BGP_AS_TRANS);
 	stream_putw(s, send_holdtime);		/* Hold Time */
-	stream_put_in_addr(s, &peer->local_id); /* BGP Identifier */
```

Comment:
Using `peer->bgp->router_id` is not enough?

Reply:
`bgp_open_make()` API is also used to forge regular BGP open streams, and peer->local_id will need to be used. so I have to maintain this change.